microsoft-edge/origin-trials/index.md

```diff
@@ -1,26 +1,191 @@
+
+After you receive the origin trial token, use the token in one of the following two ways:
+
+
+<!-- ------------------------------ -->
+#### Option 1: A \<meta\> tag within the \<head\> element of a webpage
+
+Within the `<head>` element of any webpage, include the following `<meta>` tag:
+
+`<meta http-equiv="origin-trial" content="EXAMPLE_TOKEN">`
+
+* In the `http-equiv` attribute, use the literal string `"origin-trial"`.
+* In the `content` attribute, replace `EXAMPLE_TOKEN` by your token.
+
+
+<!-- ------------------------------ -->
+#### Option 2: An HTTP header in your HTTP server response
+
+Use an HTTP header to add an `Origin-Trial` header to your HTTP server response:
+
+`Origin-Trial: EXAMPLE_TOKEN`
+
+Replace `EXAMPLE_TOKEN` by your token.
+
+
+<!-- ====================================================================== -->
+## Submit feedback about an origin trial
+
+
+<!-- ------------------------------ -->
+#### Submit private feedback about an origin trial
+
+To submit private feedback about an origin trial to the Microsoft Edge feature team that owns the origin trial:
+
+1. Go to [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).
+
+1. Select an active trial.
+
+1. Click the **Feedback** button.
+
+
+<!-- ------------------------------ -->
+#### Submit public feedback about an origin trial
+
+The GitHub Issues UI allows public discourse about ergonomics and stability of a feature.
+
+To provide public feedback for an origin trial:
+
+1. Go to [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).
+
+1. Select an active trial.
+
+1. Click the **Issues** button next to the registration button for the origin trial.
+
+
+<!-- ====================================================================== -->
+## Renew an origin trial token
+
+Origin trial tokens expire every 6 weeks by default.  After the origin trial token expires, you must renew the token.  You can renew the token before it expires.
+
+To renew an origin trial token that has expired or is about to expire:
+
+1. Go to **My Registered Trials** and select an origin trial.
+
+1. Click **Renew**.  A fresh token is sent to you.
+
+1. Copy the new token and use it on your site; see [How to use an origin trial token](#how-to-use-the-origin-trial-token), above.
+
+
+<!-- ====================================================================== -->
+## Allowed origins
+
+Edge origin trials only support SSL-enabled domains. For localhost testing of a feature, please use the appropriate Edge Feature Flag located in `edge://flags`. In the future, we may publish public tokens that are valid for localhost testing.
+
+
+<!-- ====================================================================== -->
+## Subdomains, URL paths, and query parameters
```

Comment:
move up to bottom of Register section, where this info is needed

Reply:
i integrated/moved info about how to fill in the form, into the steps for Registering
@leahmsft